.github/styles/kong/Terms.yml

```diff
@@ -0,0 +1,17 @@
+extends: substitution
+message: Use '%s' instead of '%s'.
+level: error
+ignorecase: false
```

Comment:
I wonder if we should have two substitution lists: one that ignores case, and one that's specifically there to fix case. I can't get the `[Bb]` syntax to work currently - I don't know that it's supposed to work in a key?